src/message.ts

```diff
@@ -613,6 +625,49 @@ export class Message implements Sayable {
   // }
 
   /**
+   * @param sendTo Room/Contact
+   */
+  public forward(room: Room): Promise<boolean>
+  public forward(contact: Contact): Promise<boolean>
+  public forward(sendTo: Room|Contact): Promise<boolean> {
+    const m = <MsgRawObj>this.rawObj
```

Comment:
That is because `this.rawObj` might be `undefined`.

So you need to check `undefined` instead of force it to be `MsgRawObj`.

Add the follow code at the begining of the function, then you will be able to use `this.rawObj` because that will make sure it is not `undefined`.

```ts
if (!this.rawObj) {
  throw new Error('no rawObj!')
}
```